caikit/runtime/servicers/global_train_servicer.py

```diff
@@ -360,6 +360,19 @@ def _train_and_save_model_subproc(cls, *args, **kwargs):
         proc.start()
         proc.join()
 
+        if proc.exitcode is not None:
+            # If process exited with a non-zero exit code
+            if proc.exitcode == 137:
```

Comment:
This feels pretty platform specific. Is there any platform-agnostic way to detect different exit codes? Since the process is `python` itself, I would think maybe `sys` would be able to do something like this? Just a guess

Reply:
There doesn't seem to be a straight forward way to get platform independent OOM exit code.  `sys` doesn't seem to have declaration of exit codes, `os` have some, but not for OOM case and neither in `signal` :/